doc/Release_Build_Creation.rst

```diff
@@ -4,6 +4,15 @@ Release Build Creation
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
+***************
+Prerequisites
+***************
+Your system needs to fulfill the following requirements in order to successfully create the CORTX release build.
+
+- CentOS 7.8.2003
+- At least 3 network interfaces
+- At least 2 disks
```

Comment:
I hope, it should be total 4 disks

Reply:
I thought minimum 2 disks and 3 network interfaces as that is required in the config.ini. Please correct me if this assumption is incorrect.
![image](https://user-images.githubusercontent.com/67787582/118591614-0b473600-b7d7-11eb-82bb-e090b824ce1e.png)

Reply:
OK, makes but this information is also being updated in https://github.com/Seagate/cortx/pull/1048/files#diff-7db77d583ae18894660033331355adb36c94332f71912c5a794357fbf43b5fac

Reply:
OK, makes but this information is also being updated in https://github.com/Seagate/cortx/pull/1048/files#diff-7db77d583ae18894660033331355adb36c94332f71912c5a794357fbf43b5fac

Reply:
Your modifications look much better. Should we move the this section below to this **Release Build Creation** page instead? Since user need to complete the **Release Build Creation** first before proceeding to **Provision Release Build**.

![image](https://user-images.githubusercontent.com/67787582/118599633-6f232c00-b7e2-11eb-846e-af4c1df37308.png)

Reply:
Also, we should update the info as per recommended,

![image](https://user-images.githubusercontent.com/65269273/118600596-d6cf7c00-b7ce-11eb-99fc-0b4eded2a8b3.png)

Reply:
yes, sounds good